CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 ## [Unreleased]
 
 
+## [0.25.0] - 2019-04-05
+### Changed
+- [Tendermint] Upgraded to 0.31.2
+- [IAVL] upgraded to 0.12.2
+- [Config] Tendermint.TimeoutFactor moved to Execution.TimeoutFactor (and reused for NoConsensus mode)
+- [Kernel] Refactored and various exported methods changed
+
+### Added
+- [CLI] Burrow deploy can now run jobs in parallel according to their dependency structure
```

Comment:
That's not in place yet. All there is that burrow deploy can execute multiple burrow deploy files, and execute them in parallell.